wig/compare.go

```diff
@@ -76,43 +76,32 @@ func SortByCoord(w []Wig) {
 
 // Pearson calculates the Pearson Correlation Coefficient between two input wig slices. Data values equal to the 'missing' value are ignored.
```

Comment:
not really a big deal, but since the wigs aren't stored as slices anymore, the godoc wording could be updated